packages/commerce-sdk-react/src/hooks/ShopperCustomers/mutation.test.tsx

```diff
@@ -0,0 +1,233 @@
+/*
+ * Copyright (c) 2022, salesforce.com, inc.
+ * All rights reserved.
+ * SPDX-License-Identifier: BSD-3-Clause
+ * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+import React from 'react'
+import path from 'path'
+import {mockHttpResponses, renderWithProviders, queryClient} from '../../test-utils'
+import {fireEvent, screen, waitFor} from '@testing-library/react'
+import {ShopperLoginHelpers, useShopperLoginHelper} from '../ShopperLogin'
+import {useShopperCustomersMutation, ShopperCustomersMutations} from './mutation'
+import nock from 'nock'
+
+const {withMocks} = mockHttpResponses({directory: path.join(__dirname, '../../../mock-responses')})
+
+const CUSTOMER_EMAIL = 'kobe@test.com'
+const CUSTOMER_ID = 'abkehFwKoXkbcRmrFIlaYYwKtJ'
+const RANDOM_STR = Math.random()
```

Comment:
Do we really need this random string? It looks like we're not using it in any of the test assertions.

Reply:
Correct. We don't need it we don't want to test mutations at the response data level.